README.md

```diff
@@ -136,10 +136,16 @@ After installation open `cmd.exe` and verify `python --version` returns current
 #### DUNE installation on Windows <a name="dune-windows"></a>
 
 1. Install [Chocolatey](https://docs.chocolatey.org/en-us/choco/setup).
-2. In PowerShell run as administrator following command:
-`choco install antelopeio-dune --version=1.1.0`
-3. Restart your computer (this is because %PATH% has to be reloaded. In cmd.exe it is enough to run command `refreshenv`).
-4. Open PowerShell / cmd.exe and try that following command works: `dune.bat --version`.
+2. Download latest `*.nupkg` from [latest release](https://github.com/AntelopeIO/DUNE/releases/latest) page.
+3. Run PowerShell as administrator and go to the directory where your *`.nupkg` was downloaded.
+4. Run following commands:
+```
+choco uninstall antelopeio-dune -y
+choco install python docker-desktop -y
+choco install .\antelopeio-dune.1.1.0.nupkg -y
```

Comment:
The issue is that Chocolatey cannot install dependencies if you install package from the local nupkg file.
I'm waiting for Chocolatey team to review and approve DUNE 1.0.0 (previous version).
Once they do this I will upload new DUNE 1.1.0 to Chocolatey repo and new step to install dune will be simpler (it will automatically install dependencies):
`choco install antelopeio-dune`